oci-seccomp-bpf-hook.go

```diff
@@ -319,31 +340,47 @@ func runBPFSource(pid int, profilePath string, log *logrus.Logger) error {
 
 	perfMap.Stop()
 	log.Println("PerfMap Stop")
-	if err := generateProfile(syscalls, profilePath); err != nil {
+	if err := generateProfile(syscalls, profilePath, inputFile); err != nil {
 		return err
 	}
 	return nil
 }
 
 // generate the seccomp profile from the syscalls provided
-func generateProfile(c map[string]int, fileName string) error {
+func generateProfile(calls map[string]int, fileName string, inputFile string) error {
 	s := types.Seccomp{}
+	inputProfile := types.Seccomp{}
+
+	if inputFile != "" {
+		input, err := ioutil.ReadFile(inputFile)
+		if err != nil {
+			return fmt.Errorf("cannot read input profile err: %q", err)
+		}
+		err = json.Unmarshal(input, &inputProfile)
+		if err != nil {
+			return fmt.Errorf("cannot unmarshal input file err: %q", err)
+		}
+	}
+
 	var names []string
-	for s, t := range c {
+	for s, t := range calls {
```

Comment:
What is an s and a t?  Could you spell it out.  Debugging this after the fact would be better with multiple characters describing the code.